modules/Oculus_module/src/OculusModule.cpp

```diff
@@ -1122,10 +1162,13 @@ bool OculusModule::updateModule()
     } else if (m_state == OculusFSM::Configured)
     {
         // check if it is time to prepare or start walking
-        float buttonMapping;
+        float buttonMapping = 0.0;
```

Comment:
Probably, it is safer to initialize it with `-1.0`, since we have later a check if the value is bigger than zero.